src/server_manager/web_app/ui_components/outline-about-dialog.js

```diff
@@ -19,46 +19,12 @@ import '@polymer/paper-dialog/paper-dialog.js';
 import './cloud-install-styles.js';
 import {Polymer} from '@polymer/polymer/lib/legacy/polymer-fn.js';
 import {html} from '@polymer/polymer/lib/utils/html-tag.js';
+import {unsafeHtml} from '../../infrastructure/polymer_html';
+import style from './outline-about-dialog.css';
+
 Polymer({
   _template: html`
-    <style include="cloud-install-styles">
-      #dialog {
-        width: 80%;
-        text-align: center;
-      }
-
-      #outlineLogo {
-        height: 100px;
-        margin-top: 20px;
-      }
-
-      #version {
-        font-weight: bold;
-      }
-
-      p {
-        margin: 20px 0 0 0;
-        text-align: left;
-      }
-
-      #version {
-        margin: 0;
-        text-align: center;
-      }
-
-      a {
-        color: #00ac9b;
-      }
-
-      #licenses {
-        min-width: 90%;
-      }
-
-      #licenses code {
-        font-size: 0.7em;
-      }
-    </style>
-
+    ${unsafeHtml(`<style include="cloud-install-styles">${style}</style>`)}
```

Comment:
Why do we need the `unsafeHtml` function? I see it creates a template HTML element with the css. Could we just inject `${style}`? If not, it may be worth adding a comment in the function definition.